src/pages/404.js

```diff
@@ -21,13 +21,12 @@ const PageNotFound = ({location}: Props) => (
     <Container>
       <div css={sharedStyles.articleLayout.container}>
         <div css={sharedStyles.articleLayout.content}>
-          <Header>Page Not Found</Header>
-          <TitleAndMetaTags title="React - Page Not Found" />
+          <Header>Страница не найдена</Header>
+          <TitleAndMetaTags title="React - Страница не найдена" />
           <div css={sharedStyles.markdown}>
-            <p>We couldn't find what you were looking for.</p>
+            <p>Мы не смогли найти то, что вы искали.</p>
             <p>
-              Please contact the owner of the site that linked you to the
-              original URL and let them know their link is broken.
+             Пожалуйста, свяжитесь с владельцем сайта с которого вы перешли по URL адресу и расскажите что ссылка битая.
```

Comment:
Как вам такой вариант перевода второй части предложения?
```suggestion
             Пожалуйста, свяжитесь с владельцем сайта, с которого вы перешли по URL адресу, и уведомите их о битой ссылки.
```

Reply:
Насчет запятых я тоже думаю, что по правилам они нужны, но они утяжеляют несколько, а читается вроде легко без пауз тоже. Можем исключение?

«Расскажите» менее формально чем «уведомите», поэтому думаю лучше оставить.
«Их» множественное, а владелец в единственном числе.

«расскажите что ссылка битая» и «расскажите о битой ссылке», по-моему равнозначны